pkgs/by-name/fr/freedownloadmanager/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  kdePackages,
+  gtk3,
+  atk,
+  cairo,
+  gdk-pixbuf,
+  pango,
+  openssl,
+  icu,
+  mysql80,
+  libdrm,
+  autoPatchelfHook,
+  patchelf,
+  desktop-file-utils,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "freedownloadmanager";
+  version = "6.26.1.6177";
+
+  src = fetchurl {
+    url = "http://debrepo.freedownloadmanager.org/pool/main/f/freedownloadmanager/freedownloadmanager_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-yjyRB/4vnVXB8ZmicZxsrENCCUtp7bRR0YY5x0JtmTg=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+    patchelf
+    kdePackages.wrapQtAppsHook
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    kdePackages.qtbase
+    kdePackages.qtdeclarative # QML
+    kdePackages.qtmultimedia
+    kdePackages.qtwayland
+    kdePackages.qt5compat
+
+    # GTK3 dependencies (likely for QPA platform theme like qt6gtk2)
+    gtk3
+    atk
+    cairo
+    gdk-pixbuf
+    pango
+
+    openssl
+    icu
+    mysql80 # Provides libmysqlclient.so for the bundled Qt SQL plugin
+
+    libdrm
+  ];
+
+  preFixup = ''
```

Comment:
Nit: you could move preFixup below installPhase as it happens after installPhase is done.

Reply:
Yes, but to make the file more linear anyways for clarity it could be said that putting preFixup lower in the file is more readable. Order doesn't matter in an attrset, but you wouldn't put pname last, no? Just a nit tho.